macos/FluentUI/Button/Button.swift

```diff
@@ -388,7 +388,7 @@ class ButtonCell: NSButtonCell {
 			return NSZeroRect
 		}
 
-		let isRTL = NSApp.userInterfaceLayoutDirection == .rightToLeft
+		let layoutDirectionSign : CGFloat = NSApp.userInterfaceLayoutDirection == .rightToLeft ? -1 : 1
```

Comment:
for reference: 

https://developer.apple.com/documentation/appkit/nsview/1483254-userinterfacelayoutdirection

Reply:
The type specifier is helpful because Swift infers `int` based on the literals, and complains later when I try to use this value in CGFloat calculations.  Sometimes I think language designers need to go back to math class and forget the bad habits they learned from C...